accumulo-store/src/main/java/gaffer/accumulostore/key/core/AbstractCoreKeyIteratorSettingsFactory.java

```diff
@@ -48,7 +49,7 @@ public IteratorSetting getElementFilterIteratorSetting(final View view, final Ac
     }
 
     @Override
-    public IteratorSetting getAggregatorIteratorSetting(final AccumuloStore store) throws IteratorSettingException {
+     public IteratorSetting getAggregatorIteratorSetting(final AccumuloStore store) throws IteratorSettingException {
```

Comment:
The only difference here is that a space has been added at the start of the line - it should be deleted.